trailblazer/store/models.py

```diff
@@ -89,6 +89,7 @@ class Analysis(Model):
     ticket_id = Column(types.String(32))
     uploaded_at = Column(types.DateTime)
     workflow_manager = Column(types.Enum(*WorkflowManager.list()), default=WorkflowManager.SLURM)
+    order_id = Column(types.Integer, index=True)
```

Comment:
I think we want it for most that we filter on? I am not a database expert, though.

Reply:
Can it not be that multiple different analysis have the same order id, I don't have the overview of this so I do not know. 

I think in this table the analysis id (primary key) is also an index for this table. One thing to consider would be that creating an index creates a pointer table that needs to be updated. 

So i guess we nee to consider whether an index on the ID column will give us a tangible increase in query speed.

Reply:
Yes, multiple analyses can have the same order id, and for the new order view that is being developed we want to fetch these analyses. So this would result in a query like "SELECT * FROM Analysis a WHERE a.order_id == <order_id>". 

Upon discussion we will start out with an index and then evaluate if we run into problems.